one-ui/src/test/java/com/marklogic/hub/oneui/TestHelper.java

```diff
@@ -188,11 +190,35 @@ private void assignRoleToUser(String username, String role) {
         user.save();
     }
 
-    protected GenericDocumentManager getFinalGenericDocumentManager(DatabaseKind databaseKind) {
-        return getFinalClient(databaseKind).newDocumentManager();
+    protected EvalResultIterator runJsInDatabase(String query, String databaseName) {
+        try {
+            return getServerEval(databaseName).javascript(query).eval();
+        }
+        catch(FailedRequestException e) {
+            logger.error("Failed run code: " + query, e);
+            e.printStackTrace();
+            throw e;
```

Comment:
This is a lot of redundant logging for an exception. If the point is to make it clear what query failed, just rethrow the exception:

    throw new RuntimeException("Failed to run query:  " + query,e);